### DISCUSSION TITLE
remote mysql database

### DISCUSSION THREAD

**Discussion by simonphughes** [spaceuptech/space-cloud#1624] [Q&A]
Just trying to do a quick POC using space cloud and think it looks great. Have quickly run up a docker-compose instance of space cloud and now would like to connect to a remote mysql database which is hosted on aws rds. Is this possible?

I'm currently using the connection string in this format, 
mysql://username:password@myinstance-instance-1.xxxxxxxxxx.eu-west-2.rds.amazonaws.com:3306/myDBName

but getting the following error.
default addr for network 'myinstance-instance-1.xxxxxxxxxx.eu-west-2.rds.amazonaws.com:3306' unknown

Any help greatly appreciated, and Yes that instance of RDS has a public endpoint and the appropriate rules in place to be accessed by my PC which is running the docker image.

**Comment by YourTechBud** [comment]
That's surprising. Technically it should just work. Can we confirm if our containers have internet connectivity?

Maybe try using netcat from within a container to connect to your remote database?
